www/js/controllers.js

```diff
@@ -26,6 +26,31 @@ angular.module('smartAlarm.controllers', [])
 
 })
 
+.controller('DestinationTimeCtrl', function ($scope, ionicTimePicker, $ionicPopup) {
```

Comment:
This controller was removed from master on Thursday.  I am not sure why it is still here.  Please remove the entire DestinationTimeCtrl from your branch please.